hack/update-apidocs.sh

```diff
@@ -47,5 +47,4 @@ EOF
     xargs -0 go run ./hack/apidoc-gen/main.go -section-link="#${version}" >> "${docfile}"
 }
 
-genVersionedDoc "v1beta1"
```

Comment:
@kron4eg Since we shouldn't be making changes to the v1beta1 API anymore. I have disable API docs generation for it.